common/CMakeLists.txt

```diff
@@ -171,9 +171,45 @@ set(kissfft_srcs
   src/fft/kiss_fftr.c
 )
 
+set(executor_incs
+  include/pcl/experimental/executor/executor.h
+  include/pcl/experimental/executor/property.h
+  include/pcl/experimental/executor/type_trait.h
+)
+
+set(executor_default_incs_impl
+  include/pcl/experimental/executor/default/cuda_executor.hpp
+  include/pcl/experimental/executor/default/omp_executor.hpp
+  include/pcl/experimental/executor/default/inline_executor.hpp
+  include/pcl/experimental/executor/default/sse_executor.hpp
+)
```

Comment:
what's with the `default` folder encapsulation? Why do you feel it is needed?

Reply:
No such need for it, it was mostly there to keep a cleaner structure.
The default folders just indicate the default executors provided by PCL.

Reply:
I'm struggling to see the use case to keep the folder. what other use cases do you envision that really benefit from the existence of the `default` folder?

Reply:
There is no benefit. Will remove it while integrating v0.3

Reply:
Removed